hazelcast/src/test/java/com/hazelcast/jet/core/Processors_slidingWindowingIntegrationTest.java

```diff
@@ -185,14 +185,17 @@ public boolean complete() {
             if (!emitFromTraverser(traverser)) {
                 return false;
             }
-            if (!complete) {
-                try {
-                    Thread.sleep(Long.MAX_VALUE);
-                } catch (InterruptedException e) {
-                    // proceed to returning true
-                }
-            }
-            return true;
+            return complete;
+//            if (!complete) {
+//                return false;
+//                try {
+//                    Thread.sleep(Long.MAX_VALUE);
+//                } catch (InterruptedException e) {
+//                     proceed to returning true
+//                }
+//            } else {
+//                return true;
+//            }
```

Comment:
Did we remove it because it blocked the job cancellation? If so, we can remove the commented-out code.